File: 0101_0200/0156/0156.go
package main

import (
	"fmt"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var n, k int
	kf, nf := 1, 1
	fmt.Fscan(f, &n, &k)
	for i := 1; i <= k; i++ {
		kf *= i
		nf *= n - i + 1
	}
	fmt.Fprint(q, nf*nf/kf)
}

File: 0101_0200/0156/0156.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    n, k = map(int, f.read().split())
    nf, kf = 1, 1
    for i in range(1, k + 1):
        kf *= i
        nf *= n - i + 1
    q.write(str(nf * nf // kf))
